blacken_docs.py

```diff
@@ -36,6 +36,13 @@
     r'(?P<after>^(?P=indent)\\end{minted}\s*$)',
     re.DOTALL | re.MULTILINE,
 )
+PYTHONS = r'(\{minted\}|\{minted\}\{python\}|\{(pyblock|pycode|pyconsole|pyverbatim)\})'
```

Comment:
you could have two separate ones, `LATEX_START` and `LATEX_END`

Reply:
That is a good idea. Still I am not really happy. How to guarantee that a pycode start is matched with a pycode end? By giving options, the parser will accept a \begin{pycode} a = 3 \end{pyverbatim}. Of course, tex will tackle this, but how to ensure that blacken-docs will not mess up? Actually, I would build the thing with a match for each separate type of environment (but my python skills are not yours, so I might be very much mistaken here.)

Reply:
oh hmm, yeah -- you could use regex backreferences -- if you leave this open I can modify this PR to get that to work and credit you with the patch 👍